app/src/main/java/org/literacyapp/battery/BatteryReceiver.java

```diff
@@ -0,0 +1,32 @@
+package org.literacyapp.battery;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.BatteryManager;
+import android.util.Log;
+import android.widget.Toast;
+
+
+public class BatteryReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+
+        int level = intent.getIntExtra(BatteryManager.EXTRA_LEVEL, -1);
+        int scale = intent.getIntExtra(BatteryManager.EXTRA_SCALE, -1);
+        float batteryPct = level / (float)scale;
+
+        Log.d("Battery level: ", String.valueOf(level));
+        Log.d("Battery scale: ", String.valueOf(scale));
+        Log.d("Battery status: ", String.valueOf(batteryPct));
+
+        if(batteryPct <= 10 || batteryPct != 1.0) {
```

Comment:
@jogrimst When the device is connected batteryPct changes to 1.0 and the activity is launch, so to ignore that I check this possibility.